packages/x-charts/src/context/CartesianProvider/computeValue.ts

```diff
@@ -29,26 +29,13 @@ const getRange = (drawingArea: DrawingArea, axisName: 'x' | 'y', isReverse?: boo
 
 const zoomedScaleRange = (scaleRange: [number, number] | number[], zoomRange: [number, number]) => {
   const rangeGap = scaleRange[1] - scaleRange[0];
-  const rangeMinMultiplier = zoomRange[0] / 100;
-  const rangeMaxMultiplier = zoomRange[1] / 100;
-  const min = (scaleRange[0] - rangeGap * rangeMinMultiplier) / (1 - rangeMinMultiplier);
```

Comment:
I restarted from scratch. But seems the error come from the `/ (1 - rangeMinMultiplier)` instead of ` / rangeMinMultiplier` I've no idea why it was here

Reply:
😓 I believe I was trying to offset another issue when figuring out the correct formula and kept it in...

Seems to be working now, thanks for the help 🙇